crates/bevy_pbr/src/meshlet/meshlet_bindings.wgsl

```diff
@@ -7,23 +7,19 @@
 struct PackedMeshletVertex {
     a: vec4<f32>,
     b: vec4<f32>,
-    tangent: vec4<f32>,
 }
 
-// TODO: Octahedral encode normal, remove tangent and derive from UV derivatives
 struct MeshletVertex {
     position: vec3<f32>,
```

Comment:
It's possible to do something like 16 bit unorm within the mesh aabb. idk if it would be worth it

Reply:
For what attribute? I have plans for everything except UVs.

Reply:
This comment was for position

Reply:
For position it's going to be quantized to a global grid, and then further quantized within each meshlet. Something like https://daniilvinn.github.io/2024/05/04/omniforce-vertex-quantization.html.